jane/doc/extensions/_05-kinds/intro.md

```diff
@@ -6,7 +6,7 @@ title: Intro
 
 # The kind system
 
-OxCaml includes a system of _kinds_, which classify types. Roughly, the kind
+OxCaml includes a system of _kinds_, which classifies types. Roughly, the kind
```

Comment:
This is a little ambiguous, but I prefer the original: it's the plural "kinds" that do the classifying, not the system.